test/unit/benchmark_unit_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule BenchmarkUnitTest do
```

Comment:
I think all of this has been folded into `latest` via `test/benchmarks`, no? Might have accidentally commited from your earlier changes in #28 ?